terraform/terraform.go

```diff
@@ -136,5 +136,5 @@ func UnmarshalTerraformPlannedInput(buf []byte) ([]*FastlyService, error) {
 
 func isFastlyVCLServiceResource(r *TerraformPlannedResource) bool {
 	return r.ProviderName == fastlyTerraformProviderName &&
-		(r.Type == fastlyVCLServiceType || r.Type == fastlyVCLServiceTypeV1)
+		(r.Type == fastlyVCLServiceType || r.Type == fastlyVCLServiceTypeV1) && len(r.Values.Vcl) > 0
```

Comment:
Unmarshalled resources should be a Fastly service resource even if VCL is empty.
I would suggest that we should notify the user to skip linting if VCL is empty explicitly.
So we can check VCL is empty [here](https://github.com/ysugimoto/falco/blob/main/cmd/falco/main.go#L127) and skip linting.

Reply:
Your proposal makes sense, thank you!

I have one point to discuss about how should we log the services filtered. [In the current implementation](https://github.com/ysugimoto/falco/blob/develop/cmd/falco/main.go#L119-L132), falco runs many commands other than `lint` , such as `test` and `simulate`. Only `lint` command logs which service `falco` is linting. I think there are two ways to log the services filtered;

- First, logging `{command} service of “{service name}”` also in the other commands.
- Second, filtering and logging the above like `{service of “{service name”} is filtered because that does not have VCL` in the line https://github.com/ysugimoto/falco/blob/develop/cmd/falco/main.go#L118

I feel the latter is better, but how about you?

Reply:
Sorry for the late, I also think the latter should be better, I agree with you.